package.json

```diff
@@ -28,6 +28,9 @@
     "babel-plugin-styled-components": "^1.11.0-0",
     "dotenv": "^8.2.0",
     "ganache-core": "^2.10.2",
+    "husky": "^4.2.3",
+    "prettier": "^2.0.5",
+    "pretty-quick": "^2.0.1",
```

Comment:
I wonder if we would want to use `lint-staged` instead in the future (for example, if we want to use ESLint). But it's pretty trivial to switch out.

Reply:
I actually think `lint-staged` can run `prettier`

Reply:
yeah I meant we might have to replace `pretty-quick` with `lint-staged` if we want to add ESLint later.

Reply:
Changed to `lint-staged` which will run `prettier` on staged files

Reply:
I think it's fine for now tho